biopet-utils/src/main/scala/nl/lumc/sasc/biopet/utils/FastaUtils.scala

```diff
@@ -55,4 +55,22 @@ object FastaUtils {
     if (!dictCache.contains(fastaFile)) getDictFromFasta(fastaFile)
     else dictCache(fastaFile)
   }
+
+  /** This method returns the fraction of GC for a given region */
+  def getSequenceGc(fastaFile: File, contig: String, start: Long, end: Long): Double = {
+    getSequenceGc(new IndexedFastaSequenceFile(fastaFile), contig, start, end)
+  }
+
+  /** This method returns the fraction of GC for a given region */
+  def getSequenceGc(referenceFile: IndexedFastaSequenceFile,
+                    contig: String,
+                    start: Long,
+                    end: Long): Double = {
+    require(referenceFile.isIndexed)
+    val sequence = referenceFile.getSubsequenceAt(contig, start, end)
+    val total = sequence.length()
```

Comment:
We calculated total bases based on the sum of g, c, a, t. So we excluded possible Ns from the total base counting.